components/feature/media/src/main/java/mozilla/components/feature/media/middleware/InProgressMediaMiddleware.kt

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.media.middleware
+
+import mozilla.components.browser.state.action.BrowserAction
+import mozilla.components.browser.state.action.LastAccessAction
+import mozilla.components.browser.state.action.MediaSessionAction
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.concept.engine.mediasession.MediaSession
+import mozilla.components.lib.state.Middleware
+import mozilla.components.lib.state.MiddlewareContext
+
+/**
+ * [Middleware] that updates [BrowserState.inProgressMediaTabId] everytime the user start playing media
```

Comment:
Was just about to point out that this comment needs an update, but now I wonder should we just add this logic to the `LastAccessMiddleware` which we already have now that we're using the same parent action and reducer?

Reply:
Hm thinking about this more. Ignore me :). Probably doesn't make sense to add this to the other middleware in an unrelated module. Maybe just rename `InProgressMiddleware` to `LastMediaAccessMiddleware` and update the comment?